src/main/webapp/app/view/project/project.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { Project } from 'app/shared/model/project.model';
+
+@Component({
+  selector: 'jhi-project',
```

Comment:
Ich sehe unglaublich viele JHipster prefixe. Können wir das in einem separaten Change entfernen?
Edit: Und eventuell nochmals diskutieren, ob wir nicht alles entfernen wollen, was wir nicht brauchen. Ich finde es gerade extrem schwierig schon nur die files zu durchsuchen, da es viele Komponenten hat, die durch JHipster generiert worden sind.

Reply:
ich muss die `jhi-` Prefixe im selbst geschriebenen Code natürlich entfernen..